idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -91,22 +92,25 @@ def build(self):
 
         # Mol. weights of gas - units = kg/mol. ref: NIST webbook
         mw_comp_dict = {'CH4': 0.016, 'CO2': 0.044, 'H2O': 0.018}
+        # Molecular weight should be defined in default units
+        # (default mass units)/(default amount units)
+        # per the define_meta.add_default_units method below
         self.mw_comp = Param(
                 self.component_list,
                 mutable=False,
                 initialize=mw_comp_dict,
                 doc="Molecular weights of gas components [kg/mol]",
                 units=pyunits.kg/pyunits.mol)
 
-        # Std. heat of formation of comp. - units = kJ/(mol comp) - ref: NIST
-        enth_mol_form_comp_dict = {'CH4': -74.8731, 'CO2': -393.5224,
-                                   'H2O': -241.8264}
+        # Std. heat of formation of comp. - units = J/(mol comp) - ref: NIST
+        enth_mol_form_comp_dict = {'CH4': -74.8731E3, 'CO2': -393.5224E3,
+                                   'H2O': -241.8264E3}
         self.enth_mol_form_comp = Param(
                 self.component_list,
                 mutable=False,
                 initialize=enth_mol_form_comp_dict,
-                doc="Component molar heats of formation [kJ/mol]",
-                units=pyunits.kJ/pyunits.mol)
+                doc="Component molar heats of formation [J/mol]",
```

Comment:
Minor comment, but I generally avoid putting units in doc strings now - it is effectively duplicating the information in the units and adds the possibility of the two getting out of synch.

Reply:
In the future, if we change these files back to custom units I will remove the doc strings. I left them for now since we are enforcing standard base units everywhere.